app/src/components/Map/ControlPanel.jsx

```diff
@@ -139,15 +161,11 @@ class ControlPanel extends Component {
 
   renderResume() {
     return (
-      <div className={controlPanelStyle['resume-display']}>
-        <div className={controlPanelStyle['categories-display']}>
-          <div className={controlPanelStyle['vessel-display']}>
-            <span className={controlPanelStyle['counter-description']}>vessels</span>
-            <span className={controlPanelStyle.total}>224,654</span>
-          </div>
-          <div className={controlPanelStyle['activity-display']}>
-            <span className={controlPanelStyle['counter-description']}>activity</span>
-            <span className={controlPanelStyle.total}>224,654</span>
+      <div className={controlPanelStyle['resume-display']} >
+        <div className={controlPanelStyle['categories-display']} >
+          <div className={controlPanelStyle['vessel-display']} >
+            <span className={controlPanelStyle['counter-description']} >vessels</span>
```

Comment:
Careful. You are not showing the number of vessels at all, but rather the fishing activity for a given time range, expressed in a unitless value.